### PR TITLE
add packer files for building Orka images

### PR DESCRIPTION
**Pull request by gdams**
##### Checklist

- [ ] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [ ] [faq.md](https://github.com/adoptium/infrastructure/blob/master/FAQ.md) updated if appropriate
- [ ] other documentation is changed or added (if applicable)
- [ ] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)
- [ ] VPC/QPC not applicable for this PR
- [ ] for inventory.yml changes, bastillion/nagios/jenkins updated accordingly

### CODE REVIEW

**Review by steelhead31 — APPROVED** [review]
Looks good.

File: ansible/packer/orka-base.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    }
+  }
+}
+
+variable "ORKA_TOKEN" {
+  default = env("ORKA_TOKEN")
+}
+
+variable "ORKA_ENDPOINT" {
+  default = "http://10.221.188.20"
```
Comment (karianna) [inline ansible/packer/orka-base.pkr.hcl:19]: Where is that, can we comment that or DNS it?

File: ansible/packer/orka-base.pkr.hcl [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  # Pause the provisioner until user interacts (for install Xcode etc)
+  provisioner "breakpoint" {}
+
+  # Install homebrew and ansible
+  provisioner "shell" {
+    inline = [<<EOF
+      /bin/bash -c '\
+        curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh | bash; \
```
Comment (karianna) [inline ansible/packer/orka-base.pkr.hcl:71]: Do we need to shasum that?

File: ansible/packer/orka.pkr.hcl [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    }
+  }
+}
+
+variable "ORKA_TOKEN" {
+  default = env("ORKA_TOKEN")
+}
+
+variable "ORKA_ENDPOINT" {
+  default = "http://10.221.188.20"
```
Comment (karianna) [inline ansible/packer/orka.pkr.hcl:19]: Same as before

File: ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Xcode15/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+# Xcode 15.0.1 is needed to build JDK 17+
+# Xcode 15.0.1 can be downloaded from https://developer.apple.com/download/all after authentication with apple ID and password
+# See https://github.com/adoptium/infrastructure/issues/2536#issuecomment-1708716478
+
+- name: Check if Xcode15.0.1 is installed
```
Comment (karianna) [inline ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Xcode15/tasks/main.yml:6]: Can we extract the version?

**Review by karianna — CHANGES_REQUESTED** [review]

File: ansible/packer/orka-base.pkr.hcl (hunk 1 above)
Comment (gdams) [reply ansible/packer/orka-base.pkr.hcl:19]: it's the Orka endpoint (documented in Bitwarden)

File: ansible/packer/orka.pkr.hcl (hunk 3 above)
Comment (gdams) [reply ansible/packer/orka.pkr.hcl:19]: it's the Orka endpoint (documented in Bitwarden)

File: ansible/packer/orka-base.pkr.hcl (hunk 2 above)
Comment (gdams) [reply ansible/packer/orka-base.pkr.hcl:71]: this would change all the time, it's the same way that we install homebrew on our VMs today

**Review by karianna — APPROVED** [review]

**Pull request merged**